### PR TITLE
Update RBAC comments for Istio 1.0.

### PR DESCRIPTION
**Pull request by yangminzhu**
istio/istio#6377

### CODE REVIEW

File: rbac/v1alpha1/rbac.proto [hunk 1]
```diff
@@ -192,21 +166,18 @@ message ServiceRoleBinding {
...
-// the "properties" in the "subject" part of the "authorization" template
-// (https://github.com/istio/istio/blob/master/mixer/template/authorization/template.proto).
+// Subject defines an identity. The identity is either a user or identified by a set of `properties`.
+// The supported keys in `properties` are listed in "constraint and properties" page.
 message Subject {
   // Optional. The user name/ID that the subject represents.
   string user = 1;
 
   // Optional. The group that the subject belongs to.
+  // [#not-implemented-hide:]
```
Comment (geeknoid) [inline rbac/v1alpha1/rbac.proto]: To hide this from the generated docs, you need to use $hide_from_docs instead. You can see what is generated in the pb.html file.

Comment (yangminzhu) [reply rbac/v1alpha1/rbac.proto]: Done. (It seems `[#not-implemented-hide:]` also works)

File: rbac/v1alpha1/rbac.proto [hunk 2]
```diff
@@ -232,10 +203,10 @@ message RoleRef {
...
 // ```yaml
-// apiVersion: "config.istio.io/v1alpha1"
+// apiVersion: "config.istio.io/v1alpha2"
 // kind: RbacConfig
 // metadata:
-//   name: istio-rbac-config
+//   name: default
 //   namespace: istio-system
 // spec:
 //   mode: ON_WITH_INCLUSION
```
Comment (liminw) [inline rbac/v1alpha1/rbac.proto]: Missing "inclusion" parameter here?

Comment (yangminzhu) [reply rbac/v1alpha1/rbac.proto]: I think it's just hided from the github page, you can find it here: https://github.com/istio/api/blob/f39acdf001b3a425a862a5c23f9e591daddccdab/rbac/v1alpha1/rbac.proto#L213

**Pull request merged**